adapters/MythicMobsAdapter/src/main/kotlin/com/caleb/typewriter/mythicmobs/entries/action/DespawnMobActionEntry.kt

```diff
@@ -0,0 +1,34 @@
+package com.caleb.typewriter.mythicmobs.entries.action
+
+import io.lumine.mythic.bukkit.BukkitAdapter
+import io.lumine.mythic.bukkit.MythicBukkit
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.Criteria
+import me.gabber235.typewriter.entry.Modifier
+import me.gabber235.typewriter.entry.entries.ActionEntry
+import me.gabber235.typewriter.utils.Icons
+import org.bukkit.entity.Player
+import java.util.*
+
+
+@Entry("despawn_mythicmobs_mob", "Despawn a mob from mythicmobs", Colors.ORANGE, Icons.HANDSHAKE)
```

Comment:
Handshake feels like a weird Icon for this. Have a look at [font awesome icons](https://fontawesome.com/search?o=r&m=free) to see the available icons (only the free ones).